README.md

```diff
@@ -21,19 +19,9 @@ Latest Stable Release  [![Release Version](https://img.shields.io/github/release
 Latest Development Release  [![Release Version](https://img.shields.io/github/release/espressif/arduino-esp32/all.svg?style=plastic)](https://github.com/espressif/arduino-esp32/releases/latest/) [![Release Date](https://img.shields.io/github/release-date-pre/espressif/arduino-esp32.svg?style=plastic)](https://github.com/espressif/arduino-esp32/releases/latest/) [![Downloads](https://img.shields.io/github/downloads-pre/espressif/arduino-esp32/latest/total.svg?style=plastic)](https://github.com/espressif/arduino-esp32/releases/latest/)
 
 
-### Installation Instructions
-- Using Arduino IDE Boards Manager (preferred)
-  + [Instructions for Boards Manager](docs/arduino-ide/boards_manager.md)
-- Using Arduino IDE with the development repository
-  + [Instructions for Windows](docs/arduino-ide/windows.md)
-  + [Instructions for Mac](docs/arduino-ide/mac.md)
-  + [Instructions for Debian/Ubuntu Linux](docs/arduino-ide/debian_ubuntu.md)
-  + [Instructions for Fedora](docs/arduino-ide/fedora.md)
-  + [Instructions for openSUSE](docs/arduino-ide/opensuse.md)
-- [Using PlatformIO](docs/platformio.md)
-- [Building with make](docs/make.md)
-- [Using as ESP-IDF component](docs/esp-idf_component.md)
-- [Using OTAWebUpdater](docs/OTAWebUpdate/OTAWebUpdate.md)
+### Documentation
+
+You can use [Arduino-ESP32 Online Documentation](https://docs.espressif.com/projects/arduino-esp32/en/docs-structure/index.html) to get all information about this project.
```

Comment:
I will mention also Getting started here. Maybe it will be reasonable to have it as Separate section for newcomers.

Reply:
We can add more links to the docs here.

- Getting Started
- Installing (Windows, Linux and macOS)
- Libraries
- ESP-IDF as Component
- FAQ
- Troubleshooting